ui/src/components/form/QForm.js

```diff
@@ -113,7 +113,7 @@ export default Vue.extend({
       this.validateIndex++
 
       this.getValidationComponents().forEach(comp => {
-        comp.resetValidation()
+        typeof comp.resetValidation === 'function' && comp.resetValidation()
```

Comment:
not having resetValidation() would be a user-error. This is required.

Reply:
There is no documentation I can find about it being required. Even we have in dev at least one case like this.
And I can think of some reasons not to implement it :)
The problem is that if it's not present it all crashes and it's not easy to see why.

Reply:
As this is something seldom called , I think the benefits of this being checked are worth it.